index.js

```diff
@@ -115,77 +115,251 @@ function initCarousel(options) {
 }
 
 var plugins = {
-  customCarousel: document.querySelectorAll(".js-carousel-food-spin")
+  customCarousel: document.querySelectorAll('.js-carousel-food-spin')
 };
 
 document.addEventListener("DOMContentLoaded", function () {
   if (plugins.customCarousel.length) {
     for (var i = 0; i < plugins.customCarousel.length; i++) {
       var carousel = initCarousel({
         node: plugins.customCarousel[i],
-        speed: plugins.customCarousel[i].getAttribute("data-speed"),
+        speed: plugins.customCarousel[i].getAttribute('data-speed'),
       });
     }
   }
 });
 
-//color change//
 const prevBtn = document.querySelector('.js-prev');
 const nextBtn = document.querySelector('.js-next');
 const ellipse = document.querySelector('.js-food-spin-ellipse');
 const button = document.querySelector('.js-food-spin-order-button');
 const price = document.querySelector('.js-food-spin-price');
 const title = document.querySelector('.js-food-spin-title');
 const description = document.querySelector('.js-food-spin-description');
-const items = [...document.querySelectorAll('.js-food-spin-item')];
+const pagination = document.querySelector('.js-carousel-food-spin-pagination');
+const menuItems = [...document.querySelectorAll('.js-food-spin-item')];
+const ItenSubmenu = [...document.querySelectorAll('.js-food-spin-submenu')];
+const dishesPlatos = [...document.querySelectorAll('.js-carousel-food-spin-items')];
+const colorsvg = document.querySelector('.js-food-spin-svg');
+const modalventana = document.querySelector('.js-ventana-Modal');
```

Comment:
Por cierto, a las variables tambien aplica lo de poner en mayuscula la letra inicial de las palabras. A las clases js no les aplica eso
```suggestion
const modalVentana = document.querySelector('.js-ventana-modal');
```